.github/workflows/chart-bumper.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: 3.7
+          python-version: 3.13.1
```

Comment:
This is needed because Python 3.7 has been deprecated and GitHub Actions no longer support it https://github.com/caraml-dev/helm-charts/actions/runs/12367949105/job/34517086962#step:4:11